.github/ISSUE_TEMPLATE/BC_Break.md

```diff
@@ -1,6 +1,8 @@
 ---
 name: 💥 BC Break
 about: Have you encountered an issue during upgrade? 💣
+title: "[BC BREAK]: [DESCRIPTION]"
```

Comment:
Copy & paste from Slack:

```
I wonder if we want to use square brackets for the description in #10 as I think it indicates a "placeholder" where the user has to put the real description in.
So does that mean, that he has to remove the square brackets from BC BREAK aswell and replace that text?

And since we initially adding labels anways, I dont think we need the prefix as the label is right next to the description in the issue overview?

so probably we can omit [BC BREAK] at all as the label indicates that after issue created
```